components/gc-featured-link/gc-featured-link-fr.html

```diff
@@ -180,6 +180,21 @@ <h3>Modèles de conception précédents</h3>
 		</code></pre>
 	</details>
 
-	<h2>Évaluation et rapport</h2>
-	<p>Il n'y a pas d'évaluation et de rapport disponibles pour ce composant.</p>
+	<div lang="en">
+	<h2>Evaluation and report</h2>
+	<ul>
+		<li>Accessibility pre-assessment #1
+			<ul>
+				<li><a href="./reports/a11y-1-en.html" hreflang="en"><span class="wb-inv">Pre-assessment report #1</span> English report</a></li>
+				<li><a href="./reports/a11y-1-fr.html" hreflang="fr"><span class="wb-inv">Pre-assessment report #1</span> French report</a></li>
+			</ul>
+		</li>
+		<li>Accessibility assessment #1
+			<ul>
+				<li><a href="./reports/a11y-2-en.html" hreflang="en"><span class="wb-inv">Assessment report #1</span> English report</a></li>
+				<li><a href="./reports/a11y-2-fr.html" hreflang="fr"><span class="wb-inv">Assessment report #1</span> French report</a></li>
+			</ul>
+		</li>
+	</ul>
+</div>
```

Comment:
Those report link need also to be added in the index.json-ld under "pages/examples/reports" path. 

Like this: https://github.com/wet-boew/GCWeb/blob/master/sites/footers/index.json-ld#L135-L146

Reply:
done